pset-functions.js

```diff
@@ -174,6 +276,25 @@ Use the following test cases to confirm your program meets the success criteria
 console.log('Problem 4:')
 
 // Add your code below this line
+let problem4test = "moon"
+console.log(`Test Word: ${problem4test}`)
+
+function isPalindrome(word){
```

Comment:
@kertcodes nice! code is easy to follow, only comment is that you should adhere to the best practice syntax for `if..else` statements

```
if (condition) {

} else {

} 
```